frontend/src/containers/App.jsx

```diff
@@ -1,17 +1,35 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import Navbar from "../components/Navbar"
 import CurrentWeather from "./CurrentWeather"
 import PersonDetails from "./PersonDetails"
 import BeneficialOwners from "./BeneficialOwners"
+import { getDataspaceConfiguration } from "../utils.js"
+import DataspaceConfigurationContext from "../context/dataspaceConfigurationContext"
 
 function App() {
+  const [dataspaceConfiguration, setDataspaceConfiguration] = useState({
+    dataspaceBaseUrl: "https://sandbox.ioxio-dataspace.com",
+    consentPortalUrl: "https://consent.sandbox.ioxio-dataspace.com",
+    definitionViewerUrl: "https://definitions.sandbox.ioxio-datasapce.com",
+    dataspaceName: "IOXIO Sandbox",
```

Comment:
The official name is `IOXIO Sandbox Dataspace`.
```suggestion
    dataspaceName: "IOXIO Sandbox Dataspace",
```